tests/test_agg_mds_mps_repo.py

```diff
@@ -0,0 +1,166 @@
+
+import json
+from mds.agg_mds.adapters import get_metadata, get_json_path_value
+from tenacity import RetryError, wait_none
+import httpx
+import respx
+
+@respx.mock
+def test_get_metadata_mps():
+    
+    json_response = {
+        'id': '23',
+        'name': 'Motif-PS-Chipshop',
+        'data_group': 'Taylor_MPS',
+        'study_types': 'CC',
+        'start_date': '2015-09-30',
+        'description': 'Motif polystyrene devices, using different flow setups. Human hepatocytes fresh isolated were used in. (3 cell model)',
+        }
+    
+    
+    field_mappings = {
+        "tags": [
+            {
+                "name":"MPS", 
+                "category":"Commons"
+            },
+            {
+                "name":"Physiological", 
+                "category":"Data Type"
+            }
+        ],
+        "authz": "",
+        "sites": "",
+        "summary": "path:description",
+        "study_url" : "path:url",
+        "location": "path:data_group",
+        "subjects": "",
+        "__manifest": "",
+        "study_name": "path:name",
+        "study_type": "path:study_types",
+        "institutions": "path:data_group",
+        "year_awarded": "",
+        "investigators": "path:data_group",
+        "project_title": "path:title",
+        "protocol_name": "",
+        "study_summary": "",
+        "_file_manifest": "",
+        "dataset_1_type": "",
+        "dataset_2_type": "",
+        "dataset_3_type": "",
+        "dataset_4_type": "",
+        "dataset_5_type": "",
+        "project_number": "",
+        "dataset_1_title": "",
+        "dataset_2_title": "",
+        "dataset_3_title": "",
+        "dataset_4_title": "",
+        "dataset_5_title": "",
+        "administering_ic": "",
+        "advSearchFilters": [],
+        "dataset_category": "",
+        "research_program": "",
+        "research_question": "",
+        "study_description": "",
+        "clinical_trial_link": "",
+        "dataset_description": "",
+        "research_focus_area": "",
+        "dataset_1_description": "",
+        "dataset_2_description": "",
+        "dataset_3_description": "",
+        "dataset_4_description": "",
+        "dataset_5_description": "",
+        "data_availability": ""
+        }
+        
+    assert get_metadata("mps", "http://test/ok", filters=None, config=None) == {}
+        
+    assert (
+            get_metadata("mps", "http://test/ok", filters=None, config={"batchSize": 256})
+            == {}
+            
+             )
+    assert (
+            get_metadata("mps", mds_url=None, filters={"study_ids": [23]}, config=None)
+            == {}
+        )
+        
+    respx.get(
+            "http://test/ok23",
+            status_code=200,
+            content=json_response,
+        )
+        
+    assert get_metadata(
+            "mps",
+            "http://test/ok",
+            filters={"study_ids": [23]},
+            mappings=field_mappings,
+            keepOriginalFields=True,
+        )=={
```

Comment:
the assertion result on the right should also have `tags`?

Reply:
hey @mfshao  the assertion had a tag but wasn't formatted --- I formatted code with black and tag should be easier to see I think?

Reply:
@mfshao Hey - am would it be possible to run the testing to see if this passes? (see above comment)